### PR TITLE
Change y-axis graduation to only show integers

### PR DESCRIPTION
**Pull request by willie-yao**
This PR changes the graduation on the y-axis of the dashboard time series to only show integers, since there can't be a decimal amount of a worker or job

### CODE REVIEW

**Comment by krancour** [comment]
@willie-yao could I possibly trouble you to split this up into multiple PRs?

It's often easier to weigh multiple changes each on their own merit instead as a group.

In this case, fixing the graduation on the y axis of the pending workloads graph is a slam dunk, but the other changes probably warrant more careful review.

File: grafana/dashboards/brigade.json [hunk 1]
```diff
@@ -6,7 +6,7 @@
   "version": 4,
   "refresh": "5s",
   "style": "dark",
-  "editable": false,
+  "editable": true,
```
Comment (krancour) [inline grafana/dashboards/brigade.json:9]: I don't think you meant to commit this change.

Comment (willie-yao) [reply grafana/dashboards/brigade.json:9]: I did not, I'm trying to revert the change but I think I messed up the git surgery

Comment (willie-yao) [reply grafana/dashboards/brigade.json]: It should be fixed now @krancour

**Review by krancour — APPROVED** [review]

**Pull request merged**